runtime/queries/typescript/highlights.scm

```diff
@@ -62,3 +62,15 @@
 
 ((identifier) @type
  (#match? @type "^[A-Z]"))
+
```

Comment:
`"override"` was also added to the list of keywords https://github.com/tree-sitter/tree-sitter-typescript/compare/3e897ea5925f037cfae2e551f8e6b12eec2a201a..6aac031ad88dd6317f02ac0bb27d099a553a7d8c#diff-d485a982e458ef8da2cc203585065b7542665cb80b78d230b1e8f77ea25825d4R33

Reply:
just pushed a fix, thanks Blaz!